$NetBSD: patch-ad,v 1.1 2009/08/31 22:00:30 he Exp $

Ignore ptyfs as well when monitoring file system usage level.

--- client/hobbitclient-netbsd.sh.orig	2006-08-09 22:09:58.000000000 +0200
+++ client/hobbitclient-netbsd.sh
@@ -20,7 +20,7 @@ uptime
 echo "[who]"
 who
 echo "[df]"
-df -P -tnonfs,kernfs,procfs,cd9660,null | sed -e '/^[^ 	][^ 	]*$/{
+df -P -tnonfs,kernfs,procfs,cd9660,null,ptyfs | sed -e '/^[^ 	][^ 	]*$/{
 N
 s/[ 	]*\n[ 	]*/ /
 }'
